src/deepsparse/transformers/pipelines_cli.py

```diff
@@ -0,0 +1,276 @@
+  token-classification},
+  --task {ner,question-answering,sentiment-analysis,text-classification,
+  token-classification}
+                        Name of the task to define which pipeline to
+                        create.Currently supported tasks dict_keys(['ner',
+                        'question-answering', 'sentiment-analysis', 'text-
+                        classification', 'token-classification'])
+  -d DATA, --data DATA  Path to file containing data for inferences, inputs
+                        should be separated via newline
+  --model-name MODEL_NAME, --model_name MODEL_NAME
+                        Canonical name of the hugging face model this model is
+                        based on
+  --model-path MODEL_PATH, --model_path MODEL_PATH
+                        Path to (ONNX) model file to run, can also be a
+                        SparseZoo stub
+  --engine-type {deepsparse,onnxruntime}, --engine_type {deepsparse,onnxruntime}
+                        inference engine name to use. Supported options are
+                        'deepsparse'and 'onnxruntime'
+  --config CONFIG       Huggingface model config, if none provided, default
+                        will be usedwhich will be from the model name or
+                        sparsezoo stub if given for model path
+  --tokenizer TOKENIZER
+                        Huggingface tokenizer, if none provided, default will
+                        be used
+  --max-length MAX_LENGTH, --max_length MAX_LENGTH
+                        Maximum sequence length of model inputs. default is
+                        128
+  --num-cores NUM_CORES, --num_cores NUM_CORES
+                        Number of CPU cores to run engine with. Default is the
+                        maximum available
+  -b BATCH_SIZE, --batch-size BATCH_SIZE, --batch_size BATCH_SIZE
+                        The batch size to use for pipelines
+  --scheduler {multi,single}
+                        The scheduler to use for the engine. Can be None,
+                        single or multi
+  -o OUTPUT_FILE, --output-file OUTPUT_FILE, --output_file OUTPUT_FILE
+                        Directs the output to a name of your choice
+####################
+Example commands:
+1) deepsparse.pipeline --task ner \
+    --model-path bert-ner-test.onnx \
+    --data input.txt
+
+2) deepsparse.pipeline --task ner \
+    --model-path models/bert-ner-test.onnx \
+    --data input.txt \
+    --config ner-config.json \
+    --output-file out.txt \
+    --batch_size 2
+
+3) deepsparse.pipeline --task sentiment-analysis \
+    --model_path models/bert-sst-test.onnx \
+    --data models/input.txt \
+    --batch_size 2 \
+    --output_file out.txt
+
+"""
+
+import argparse
+import os
+from pathlib import Path
+from typing import Optional
+
+from .loaders import SUPPORTED_EXTENSIONS, get_batch_loader
+from .pipelines import SUPPORTED_ENGINES, SUPPORTED_TASKS, pipeline
+
+
+__all__ = [
+    "cli",
+]
+
+MODEL_DIR = os.getenv("MODEL_DIR") or ""
```

Comment:
let's remove this as well as defaulting the model dir for now as discussed in slack